src/tests/tests.yaml

```diff
@@ -0,0 +1,22 @@
+charm_name: keystone-kerberos
+smoke_bundles:
+  - bionic-train
+gate_bundles:
+  - bionic-queens
+  - bionic-rocky
+  - bionic-stein
+  - bionic-train
+dev_bundles:
+  - bionic-ussuri
+  - focal-ussuri
+configure:
+- zaza.openstack.charm_tests.kerberos.setup.run_all_tests
```

Comment:
I think the run_all_tests method name is a bit misleading, as they are all really configuration steps, and it doesn't actually include the most meaningful test here (CharmKeystoneKerberosTest).

You could either rename this method, or include ALL tests. I would lean toward a rename, mainly because of the Zaza test.yaml primitives of .configure and .test.